task3/vector.js

```diff
@@ -0,0 +1,81 @@
+function sortAsc(array) {
+    return array.sort((a,b) => a-b);
+}
+
+class Vector {
+    constructor(inputArray) {
+        this.array = inputArray;
+    }
+
+    checkIsSameLength(vector) {
+        if (this.array.length !== vector.array.length) {
+            throw "Two vectors should have a same length";
+        }
+    }
```

Comment:
It rather looks like a private function which we wouldn't use outside.
We don't have private properties for Classes in JS, but in real life, we mark private methods with an underscore (like `_isPrivateMethodName`).
It, of course, doesn't prevent access from outside, but developers should understand that it's private things that couldn't be used. It's like an unspoken rule :)